zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1584,6 +1596,16 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     (e.g. the zk/myhost@EXAMPLE.COM client principal will be authenticated in ZooKeeper as zk/myhost)
     Default: false
 
+* *multiAddress.enable* :
+    (Java system property: **zookeeper.multiAddress.enable**)
```

Comment:
Same here, missing the d (internet has ruined me, no pun intended)

Reply:
true, thanks. :)